File: snippets/typescript/count_points.ts
import { QdrantClient } from "@qdrant/js-client-rest";

const client = new QdrantClient({ host: "localhost", port: 6333 });

client.count("{collection_name}", {
    filter: {
        must: [
            {
                key: "is_smoker",
                match: {
                    value: true,
                },
            },
        ],
    },
    exact: true,
});

File: snippets/go/count_points.go
package client

import (
	"context"

	"github.com/qdrant/go-client/qdrant"
)

func count() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	client.Count(context.Background(), &qdrant.CountPoints{
		CollectionName: "{collection_name}",
		Filter: &qdrant.Filter{
			Must: []*qdrant.Condition{
				qdrant.NewMatch("color", "red"),
			},
		},
	})
}
